iwyu_output.cc

```diff
@@ -240,11 +240,7 @@ OneUse::OneUse(const NamedDecl* decl, SourceLocation use_loc,
       use_loc_(use_loc),
       use_kind_(use_kind),             // full use or fwd-declare use
       in_cxx_method_body_(in_cxx_method_body),
-      comment_(comment ? comment : ""),
-      public_headers_(),
-      suggested_header_(),             // figure that out later
-      ignore_use_(false),
-      is_iwyu_violation_(false) {
```

Comment:
We shouldn't be removing `ignore_use_` and `is_iwyu_violation_` here, should we?

Won't they be garbage? I think I've seen recent claims that members are default-initialized, but I can't find a reference now. I'd prefer to keep the explicitly initialized to false anyway.